features/searches.feature

```diff
@@ -442,3 +442,13 @@ Feature: Search
     When I am on agency.gov's search page
     And I fill in "query" with "popular"
     Then I should see a suggestion to search for "popular search phrase"
+
+  Scenario: Searching with spelling suggestions
+    Given the following Affiliates exist:
+      | display_name | name       | contact_email | first_name | last_name | domains |
+      | agency site  | agency.gov | aff@bar.gov   | Jane       | Bar       | usa.gov |
+    When I am on agency.gov's search page
+    And I search for "qeury"
+    Then I should see "Showing results for query"
+    And I should see "Search instead for qeury"
+    And I should see a link to "qeury" with url that ends with "/search?affiliate=agency.gov&query=%2Bqeury"
```

Comment:
Actually, I just realized that this line describes the incorrect behavior, with the `+` prepended to the query: `%2Bqeury`.

Let's omit the `And I should see a link...` line for now, and I'll add that to the bug ticket as our failing test for future bug fixers.